README.md

```diff
@@ -121,6 +121,111 @@ connection
+| [`bytea`][psql-bytea-ref]                      | Not yet supported.|
+| [`character`][psql-character-ref]              | [`String`][java-string-ref]|
+| [`character varying`][psql-character-ref]      | [`String[]`][java-string-ref]|
+| [`cidr`] [psql-cidr-ref]                       | Not yet supported.|
+| [`circle`][psql-circle-ref]                    | Not yet supported.|
+| [`date`][psql-date-ref]                        | [`LocalDate`][java-ld-ref]|
+| [`double precision`][psql-floating-point-ref]  | [**`Double`**][java-double-ref], [`Float`][java-float-ref]|
+| [`inet`][psql-inet-ref]                        | [**`InetAddress`**][java-inet-ref]|
+| [`integer`][psql-integer-ref]                  | [**`Integer`**][java-integer-ref], [`Boolean`][java-boolean-ref], [`Byte`][java-byte-ref], [`Short`][java-short-ref], [`Long`][java-long-ref]|
+| [`interval`][psql-interval-ref]                | Not yet supported.|
+| [`json`][psql-json-ref]                        | Not yet supported.|
+| [`line`][psql-line-ref]                        | Not yet supported.|
+| [`lseg`][psql-lseq-ref]                        | Not yet supported.|
+| [`macaddr`][psql-macaddr-ref]                  | Not yet supported.|
+| [`macaddr8`][psql-macaddr8-ref]                | Not yet supported.|
+| [`money`][psql-money-ref]                      | Not yet supported.|
+| [`numeric`][psql-bignumeric-ref]               | [`BigDecimal`][java-bigdecimal-ref]|
+| [`path`][psql-path-ref]                        | Not yet supported.|
+| [`pg_lsn`][psql-pg_lsn-ref]                    | Not yet supported.|
+| [`point`][psql-point-ref]                      | Not yet supported.|
+| [`polygon`][psql-polygon-ref]                  | Not yet supported.|
+| [`real`][psql-real-ref]                        | [**`Float`**][java-float-ref], [`Double`][java-double-ref]|
+| [`smallint`][psql-smallint-ref]                | [**`Short`**][java-short-ref], [`Boolean`][java-boolean-ref], [`Byte`][java-byte-ref], [`Integer`][java-integer-ref], [`Long`][java-long-ref]|
+| [`smallserial`][psql-smallserial-ref]          | Not yet supported.|
+| [`serial`][psql-serial-ref]                    | Not yet supported.|
+| [`text`][psql-text-ref]                        | [`String[]`][java-string-ref]|
+| [`time [without time zone]`][psql-time-ref]    | [`LocalTime`][java-lt-ref]|
+| [`time [with time zone]`][psql-time-ref]       | [`LocalTime`][java-lt-ref]|
+| [`timestamp [without time zone]`][psql-time-ref]|[`LocalDateTime`][java-ldt-ref]|
+| [`timestamp [with time zone]`][psql-time-ref]  |[`ZonedDateTime`][java-zdt-ref]|
+| [`tsquery`][psql-tsquery-ref]                  | Not yet supported.|
+| [`tsvector`][psql-tsvector-ref]                | Not yet supported.|
+| [`txid_snapshot`][psql-txid_snapshot-ref]      | Not yet supported.|
+| [`uuid`][psql-uuid-ref]                        | [**`UUID`**][java-uuid-ref], [`String`][java-string-ref]||
+| [`xml`][psql-xml-ref]                          | Not yet supported. |
+
```

Comment:
We should also mention somewhere the array support as array-typed columns are supported for reads and writes.

Reply:
should I add separate section something like this or do you have any suggestion

```
Currently support the following single-dimensional arrays for both read and write

| PostgreSQL Type                                | Supported Data Type                  |
|:-----------------------------------------------|:-------------------------------------|
|[`tex[]`][psql-text-ref]                        |[`String[]`][java-string-ref]         |  
|[`integer[] or int[]`][psql-integer-ref]        |[`Integer[]`][java-integer-ref], [`Long[]`][java-long-ref], [`Short[]`][java-short-ref]|
```

Reply:
please let me know if i missed anything in array section

Reply:
@mp911de  can please let me know if anything missing here or are we good?